pkg/webui/console/views/device-general-settings/application-server-form/index.js

```diff
@@ -31,11 +31,21 @@ import { generate16BytesKey, isNonZeroSessionKey } from '@console/lib/device-uti
 
 import messages from '../messages'
 
+const m = defineMessages({
+  skip: 'Skip payload crypto',
+  include: 'Do not skip payload crypto',
+  default: 'Use default',
+})
+
 const validationSchema = Yup.object()
   .shape({
-    skip_payload_crypto: Yup.boolean().default(false),
+    skip_payload_crypto_value: Yup.string()
+      .oneOf(['skip', 'include', 'default'])
+      .required(sharedMessages.validateRequired)
+      .default('default'),
+    skip_payload_crypto_overrride: Yup.boolean().nullable().default(null),
```

Comment:
You don't need to use two values here. You can use `decode` and `encode` props of the `<Form.Field />` to marshal between notations. See my other comment below. In the validation schema you should think about validating the values in a way that they can be sent straight to the backend.